Reinstate the game archive (#107)
srobo/docs: rules/index.md

---
layout: page
title: 2020 Rules
---

2020 Rules
==========

> The rules for this year's competition will be published shortly after Kickstart.

Reviewer: The current intention is to publish them _during_ kickstart rather than after (https://github.com/srobo/tasks/issues/376)

```suggestion
The rules for this year's competition will be published during Kickstart.
```

Author: :+1: (this was blindly copied from what it said last year)

Author: https://github.com/srobo/docs/pull/107/commits/d2382cca54b19803e0deecb6dd36f7068b6b6241
